build.sh

```diff
@@ -18,17 +18,23 @@ PRINT_USAGE() {
     echo "build.sh [options]"
     echo ""
     echo "options:"
-    echo "--help, -h : Show help"
+    echo "-j          : Multicore build (i.e. -j:3 for 3 cores)"
+    echo "-icu        : Path to ICU include folder (see example below)"
+    echo "--help, -h  : Show help"
     echo "--debug, -d : Debug build"
     echo "CXX=<path to clang++>"
     echo "CC=<path to clang>"
     echo ""
-    echo "example: ./build.sh CXX=/path/to/clang++ CC=/path/to/clang"
+    echo "example : ./build.sh CXX=/path/to/clang++ CC=/path/to/clang -j:2"
+    echo "with icu: ./build.sh -icu:/usr/local/Cellar/icu4c/version/include/"
```

Comment:
Command line switch styles seem to become inconsistent. Can we consider sth. like `--CXX=... --CC=... --icu-path=...`? Similar original make style `-j [N], --jobs[=N]`, or only `--jobs=N`.